canonical/ubuntu.com
# Accessibility for disabled area on /pro/subscribe

## Done

- Disable tab key on disabled area


## QA
- Go to /pro/subscribe
- Tab through the page and check tab is not working on disabled area

## Issue / Card

Fixes https://github.com/canonical/ubuntu.com/issues/11916
